server/src/main/java/org/elasticsearch/common/geo/TriangleTreeReader.java

```diff
@@ -33,8 +33,8 @@
  * relations against the serialized triangle tree.
  */
 public class TriangleTreeReader {
+    private static final int CENTROID_HEADER_SIZE_IN_BYTES = 9;
```

Comment:
Maybe add a comment where this 9 bytes are coming from.

Reply:
I will update this in the follow-up PR when this is modified to include the centroid weight!